src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.Tests/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests.csproj

```diff
@@ -3,8 +3,7 @@
   <PropertyGroup>
     <Description>Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests Class Library</Description>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests</AssemblyName>
-    <Version>1.0.0</Version>    
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
Please undo all the changes to this file.

Reply:
Updated for this file.